build-system/amp4test.js

```diff
@@ -94,6 +99,17 @@ ${req.query.body}
   `);
 });
 
+app.use('/compose-shadow', function(req, res) {
+  const {docUrl} = req.query;
+  const viewerHtml = renderShadowViewer({
+    // Remove leading '/'.
+    src: docUrl.slice(1),
```

Comment:
Safer and self-documented:

```js
{
  src: docUrl.replace(/^\//, ''),
  // ...
}
```

Maybe we can even move that to the renderer function.

Reply:
Done. Left here for now, we can revisit if the renderer gets more popular.